pkg/gofr/http/errors.go

```diff
@@ -7,6 +7,39 @@ import (
 	"strings"
 )
 
+type MultipleErrors struct {
+	Code   int
+	Errors []error
+}
+
+func (m MultipleErrors) Error() string {
```

Comment:
What about this?

```diff
func (m MultipleErrors) Error() string { 
-         var result string 
+        var result []string
          for _, v := range m.Errors { 
-                 result += fmt.Sprintf("%s\n", v) 
+               result = append(result, fmt.Sprint(v)
         } 
  
-         return strings.TrimSuffix(result, "\n") 
+        return strings.Join(result, "\n")
 }
```

Reply:
Although both are similar implementation wise. But when I ran the benchmark tests for both these implementations, I observed that the current one (result as `string`) is `much faster` than the suggested one (result as `[]string`). Attaching the results below:

<img width="526" alt="Screenshot 2024-07-08 at 11 18 52 AM" src="https://github.com/gofr-dev/gofr/assets/44723623/915e7f54-7e68-4466-b74b-3f291f3ef4ac">

<img width="527" alt="Screenshot 2024-07-08 at 11 21 47 AM" src="https://github.com/gofr-dev/gofr/assets/44723623/7a7fec51-08a2-470d-ad69-c704f84d36ca">

Reply:
Thanks for doing the benchmark.

For me, it was more a question of code simplicity or clarification than performance.

Here we are in an error, so it occurs rarely. Performance is not the priority I think.

Anyway, I'm fine with both.